qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -47,31 +50,31 @@ def __init__(self, data, coeffs=None):
         """Initialize an operator object.
 
         Args:
-            data (PauliTable): Pauli table of terms.
+            data (Paulilist, SparsePauliOp): Pauli list of terms.
```

Comment:
Should we still support `PauliTable` as input to be backward compatible?

Reply:
Thanks. Yes. The doc string is wrong. (we still support it.)